api/src/main/java/org/pmiops/workbench/api/ConfigController.java

```diff
@@ -26,6 +27,8 @@
     WorkbenchConfig config = configProvider.get();
     return ResponseEntity.ok(
         new ConfigResponse()
+            .accessRenewalExpiry(BigDecimal.valueOf(config.accessRenewal.expiryDays))
```

Comment:
It seems we weren't using this anywhere? Should we be?

If not - do we need to pass this down at all? This config is not intended to be exhaustive of every value in the internal `WorkbenchConfig` - it only needs to include things that the client uses.